pints/_log_pdfs.py

```diff
@@ -115,6 +115,177 @@ def sample(self, n=1):
         raise NotImplementedError
 
 
+class PooledLogPDF(LogPDF):
+    r"""
+    Calculates a sum of :class:`LogPDF` objects, while pooling selected
+    parameters across log-pdfs.
+
+    This :class:`LogPDF` requires that all :class:`LogPDF` objects have the
+    same number of parameters.
+
+    This is useful for e.g. modelling the time-series of multiple individuals
+    (each individual defines a separate :class:`LogPDF`), and some parameters
+    are expected to be the same across individuals. This may be a realistic
```

Comment:
"are expected to be the same across individuals (for example, the noise parameter across different individuals within the same experiment)."

Reply:
Done